src/zc/form/browser/widgetapi.py

```diff
@@ -40,13 +40,13 @@
     _("widget-missing-multiple-value-for-edit"), "(no value)")
 
 
+@implementer(IBrowserWidget, IInputWidget)
 class BaseWidget(BrowserWidget, InputWidget):
     # Note to previous users of widgetapi:
     # .translate -> ._translate; .__prefix -> ._prefix;  NullValue ->
     # ._data_marker; .__initial_value and .__calculated_value -> replaced
```

Comment:
This seems to be the origin of the `NullValue` which was deleted in a former PR. Maybe it was forgotten to be replaced by `_data_marker`.

Reply:
I included the `._data_marker` at the respective place.